plugins/outputs/redistimeseries/redistimeseries.go

```diff
@@ -0,0 +1,106 @@
+package redistimeseries
+
+import (
+	"github.com/go-redis/redis/v7"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/common/tls"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+var sampleConfig = `
+  ## The address of the RedisTimeSeries server.
+  address = "127.0.0.1:6379"
+
+  # Redis ACL username
+  # username = ""
+  ## password to login Redis
+  # password = ""
+  database = 0
+  ## Optional TLS Config
+  # tls_ca = "/etc/telegraf/ca.pem"
+  # tls_cert = "/etc/telegraf/cert.pem"
+  # tls_key = "/etc/telegraf/key.pem"
+  ## Use TLS but skip chain & host verification
+  # insecure_skip_verify = false
+`
```

Comment:
I think you missed my comment regarding `sampleConfig`... Please change to
```suggestion
var sampleConfig = `
  ## The address of the RedisTimeSeries server.
  address = "127.0.0.1:6379"

  # Redis ACL credentials
  # username = ""
  # password = ""
  # database = 0

  ## Optional TLS Config
  # tls_ca = "/etc/telegraf/ca.pem"
  # tls_cert = "/etc/telegraf/cert.pem"
  # tls_key = "/etc/telegraf/key.pem"
  # insecure_skip_verify = false
`
```
and also adapt the README accordingly.